torchtune/modules/model_fusion/_fusion_utils.py

```diff
@@ -37,3 +37,33 @@ def fusion_params(self) -> List[str]:
         return [k for k, v in self.named_parameters()]
 
     module.fusion_params = functools.partial(fusion_params, module)
+
+
+def get_fusion_params(model: nn.Module) -> Dict[str, nn.Parameter]:
+    """
+    Return the subset of parameters from a model that correspond to fused
+    modules. Assumes that any fusion class has defined the
+    :func:`~torchtune.modules.model_fusion.FusionLayer.fusion_params` method.
+
+    Args:
+        model (nn.Module): Instance of model class containing some
+            fusion params.
+
+    Returns:
+        Dict[str, nn.Parameter]: the subset of model's state dict containing
+            only adapter parameters.
+
+    """
+    fusion_params = {}
+    for k, v in model.named_modules():
```

Comment:
could you make the for loop variables more descriptive? it's hard to follow what's happening

Reply:
This is a direct port of the get_peft_params function, so I'd prefer to keep both of those implementations in sync for now.

Reply:
The proliferation of my laziness..